File: marketdata-common/src/main/java/com/handson/dto/Quote.java
package com.handson.dto;

import com.google.gson.Gson;
import com.google.gson.GsonBuilder;

public class Quote {

    public final String code;
    public final double quote;

    public Quote(String code, double quote) {
        this.code = code;
        this.quote = quote;
    }

    public static Quote fromJson(String input) {
        return new Gson().fromJson(input, Quote.class);
    }

    public String toJson() {
        return new Gson().toJson(this);
    }

    @Override
    public String toString() {
        return toJson();
    }
}

File: CSharp/NetCore/MarketData/src/MarketDataCommon/Dto/Quote.cs
using Newtonsoft.Json;

namespace MarketDataCommon.Dto
{
    public class Quote
    {
        public string Code { get; set; }
        public double QuoteValue { get; set; }

        public Quote(string code, double quote)
        {
            Code = code;
            QuoteValue = quote;
        }

        public static Quote FromJson(string input)
        {
            return JsonConvert.DeserializeObject<Quote>(input);
        }

        public string ToJson()
        {
            return JsonConvert.SerializeObject(this);
        }

        public override string ToString() => ToJson();
    }
}
